README.md

```diff
@@ -54,25 +62,63 @@ You can combine the command line options given below.
   ./candi.sh --prefix=Your/Prefix/Path
 ```
 
-#### Multiple build processes: `[-j <N>]`, `[-j<N>]`, `[--PROCS=<N>]`
+#### Multiple build processes: `[-j <N>]`, `[--PROCS=<N>]`
 ```bash
   ./candi.sh -j <N>
 ```
 
-* Example: to use 2 build processes type `./candi.sh -j2`.
+* Example: to use 2 build processes type `./candi.sh -j 2`.
 * Be careful with this option! You need to have enough system memory (e.g. at least 8GB for 2 or more processes).
 
 ### Configuration file options
 
-Edit the configuration file behind the file "candi.cfg", e.g.
-```bash
-  gedit candi.cfg
-```
+If you want to change the set of packages to be installed,
+you can choose for a bunch of those in the configuration file
+``candi.cfg``.
+This file is a simple text file and can be changed with any text editor.
+
+Currently, we provide the packages
+
+* trilinos
+* petsc, slepc
+* superlu_dist (to be used with trilinos)
+* p4est
+* hdf5
+* opencascade
+
+for instance.
+
+There are several other options within the configuration file, e.g.
 
-You can adapt several things to your personal needs here
 * the `DOWNLOAD_PATH` folder (can be safely removed after installation),
 * the `UNPACK_PATH` folder of the downloaded packages (can be safely removed after installation),
 * the `BUILD_PATH` folder (can be safely removed after installation),
 * the `INSTALL_PATH` destination folder,
 
-and more options.
+and more.
+
+### Single package installation mode
+
+If you prefer to install only a single package, you can do so by
+```bash
+  ./candi.sh --packages="dealii"
+```
+for instance, or a set of packages by
+```bash
+  ./candi.sh --packages="opencascade petsc"
+```
+
+### Developer mode
+
+Our installer provides a software developer mode by setting
+``DEVELOPER_MODE=ON``
+within `candi.cfg`.
```

Comment:
I look for all places with only single backticks and make them to double backticks